### DISCUSSION TITLE
AttributeError: partially initialized module 'whisper' has no attribute 'load_model' (most likely due to a circular import)

### DISCUSSION THREAD

**Discussion by faizanbashir** [openai/whisper#959] [General]
I am getting the below error when I try to run the python code.

```shell
AttributeError: partially initialized module 'whisper' has no attribute 'load_model' (most likely due to a circular import)
```
Code:
```python
import whisper

model = whisper.load_model("base")
result = model.transcribe("./audio/microphone-results.wav")
print(result)
```

These are the commands for the installation:
```sh
brew install ffmpeg
pip install setuptools-rust
pip install git+https://github.com/openai/whisper.git 
```

I am able to run it as a standalone command but using the python package its failing

```sh
whisper ./audio/microphone-results.wav --language English --model tiny.en
```

**Comment by glangford** [comment]
Is your python code in a file called `whisper.py`? If so, rename it to something else.

**Reply by aivilmo** [reply]
Thank you, that works for me

**Reply by xiaoyin199** [reply]
thank

**Reply by lanbinshijie** [reply]
Thanks, works for me too.

**Reply by saw-a-bird** [reply]
I guess "whisper" is a really catchy name huh

**Reply by yaliniyalu** [reply]
Thank you

**Reply by FotieMConstant** [reply]
Thanks man, works like charm.

**Reply by Shwifty0** [reply]
Savior!

**Reply by edglazer** [reply]
One day, they will write songs about you! Many thanks!

**Reply by TroyMitchell911** [reply]
Amazing..

**Reply by wangxy4628** [reply]
救世主

**Reply by mkhuthir** [reply]
<3

**Reply by okineadev** [reply]
Thanks!

**Reply by hoangcaobao** [reply]
thanks

**Reply by TheFirstMonkey** [reply]
amazing

**Reply by woreaizhegeshijie** [reply]
thanks

**Reply by shivank22** [reply]
🗡️ Thank you

**Reply by hamzausmanghani** [reply]
My file name was `code.py`. I renamed the file to `app.py` and it worked. Thanks!

**Reply by Guo0911** [reply]
This is the seventh time I've seen this reply...

Thank you again.

**Comment by faizanbashir** [comment]
Thanks, it's working now :)

**Comment by SidraSoniaAziz** [comment]
That' works for me too.

**Comment by taaha-iqbal** [comment]
I was getting same error and I solved it using this libraries are overlapping each other I think.
pip uninstall openai_whisper
pip uninstall whisper
pip install git+https://github.com/openai/whisper.git
run python script now.
